pipelines/secure-scan.yml

```diff
@@ -34,6 +34,17 @@ steps:
   - powershell: 'get-childitem env:'
     displayName: "Print variables"
 
+  - task: EsrpMalwareScanning@1
+    displayName: 'Run ESRP Malware Scan'
+    inputs:
+      ConnectedServiceName: 'ESRP Malware Scanning service connection'
+      FolderPath: '$(Build.SourcesDirectory)\objects'
```

Comment:
>objects [](start = 45, length = 7)

can we only scan the changed folder?

Reply:
Until we can detect the changed folder reliably, I'd rather scan them all.